src/Model/Contact.php

```diff
@@ -1152,7 +1152,7 @@ public static function createFromProbe($uid, $url, $interactive = false, $networ
 		Addon::callHooks('follow', $arr);
 
 		if (empty($arr)) {
-			$result['message'] = L10n::t('The contact could not be added. It is possible you are attempting to add a contact that requires an Addon that is not enabled on this system.');
+			$result['message'] = L10n::t('The contact could not be added. It is possible you are attempting to add a contact that requires an Addon that is not configured on this system.');
```

Comment:
"The contact could not be added. Please check the relevant network credentials in your Settings -> Social Networks page."

Reply:
But possibly the addon wasn't configured by the administrator.

Reply:
Indeed, but it's less likely that the admin enabled the addon and didn't configure it than a random user just tried to add a twitter user before signing in with Twitter.